custom_python_actions/check_compliance.py

```diff
@@ -38,28 +46,81 @@ def check_readme(repo: github3.github.repo) -> bool:
         return False
 
 
+def get_team_name(code_owners: str, org_name: str) -> str:
+    code_owner_file = code_owners.strip("\n")
+    regex_output = re.findall(
+        # If anyone is a regex wizard, please help me out here!
+        r"(?=\*. *@).*(@\S*).*(@\S*)|(?=\*. *@).*(@\S*)",
```

Comment:
It looks too complicated to me and I'm afraid will also wrongly catch cases like this:
```
# * @group/user # uncomment when ready
```

Reply:
For reference, this is where I was trying things out https://regex101.com/r/q03Bd4/1 and this was my test file:

```
* @dfinity/idx
* @dfinity/idx    # some comment
*       @global-owner1 @global-owner2
*.js    @js-owner #This is an inline comment.
*/all-files @sdk-team
* @another-org/another-team # some comment
* @dfinity/idx  @dfinity/another-team
```